config/application.rb

```diff
@@ -30,9 +30,6 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 8.0
 
-    # Disable rails 7.0+ button_to behaviour
-    config.action_view.button_to_generates_button_tag = false
```

Comment:
I dug out the commit where this was added, and found this [comment thread](https://github.com/alphagov/whitehall/pull/6449#discussion_r830249277) as to why it was disabled.

Sounds as though the tests that were plaguing Benjamin no longer seem to be a concern? Great to see all this deleted!